specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -3756,6 +3844,89 @@
               "Desktop"
             ]
           }
+        }
+      }
+    },
+    "SocketAddressMatchConditionParameters": {
+      "description": "Defines the parameters for SocketAddress match conditions",
+      "required": [
+        "operator",
+        "typeName"
+      ],
+      "properties": {
+        "typeName": {
+          "type": "string",
+          "enum": [
+            "DeliveryRuleSocketAddressConditionParameters"
+          ]
+        },
+        "operator": {
+          "description": "Describes operator to be matched",
+          "type": "string",
+          "enum": [
+            "IPMatch"
+          ],
+          "x-ms-enum": {
+            "name": "SocketAddressOperator",
+            "modelAsString": true
+          }
+        },
+        "negateCondition": {
+          "description": "Describes if this is negate condition or not",
+          "type": "boolean"
+        },
+        "matchValues": {
+          "description": "The match value for the condition of the delivery rule",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "RequestClientPortMatchConditionParameters": {
+      "description": "Defines the parameters for RequestClientPort match conditions",
+      "required": [
+        "operator",
+        "typeName"
+      ],
+      "properties": {
+        "typeName": {
+          "type": "string",
+          "enum": [
+            "DeliveryRuleRequestClientPortConditionParameters"
+          ]
+        },
+        "operator": {
+          "description": "Describes operator to be matched",
+          "type": "string",
+          "enum": [
+            "Any",
+            "Equal",
+            "Contains",
+            "BeginsWith",
+            "EndsWith",
+            "LessThan",
+            "LessThanOrEqual",
+            "GreaterThan",
+            "GreaterThanOrEqual",
+            "RegEx"
```

Comment:
RegEx operator seems not available for ClientPort/ServerPort, please confirm with @jessie

Reply:
updated

Reply:
removed